IBM/carbon-addons-iot-react
# fix(image-hotspot): removed unused props

Closes #1154

**Summary**

- Removed unused props in ImageCard with hotspots causing errors

**Change List (commits, features, bugs, etc)**

- Removed `offsetX` and`offsetY` props being passed down to `Hotspot` in `ImageHotspot.jsx`
- Removed `clickToOpen` prop being passed down to `Tooltip` in `Hotspot.jsx`

**Acceptance Test (how to verify the PR)**

- Open Storybook
- Render the ImageCard with hotspots story
- Open console and see no more errors popping up
